cabal.project

```diff
@@ -0,0 +1,13 @@
+with-compiler: ghc-8.6.3
+packages:
+  dunai.cabal
```

Comment:
I'm a bit confused by the fact that this list includes pointers to a cabal file and several dirs. Is it possible to just say "." instead of `dunai.cabal`?

Reply:
the several dirs is needed so user can run `cabal build all` (for cabal version 3.0.0.0) to make cabal build all package contained in the examples folder.

it can be `./.` so it will find `.cabal` within root directory.

Reply:
Without being a cabal 3.0 expert, that seems to make more sense to me.

Reply:
it is already happened in cabal new-build style (around version 2.4)

Reply:
True. Although I never truly updated any projects to work with `cabal new-build` style in 2.4, or with any additions in 3.0. I tend to purposefully delay adopting new versions of cabal & GHC, or new and fancy tools :)